app/main.py

```diff
@@ -1,8 +1,51 @@
+def limitation(value: float,
+               min_value: float,
+               max_value: float) -> float:
+    return max(min(value, max_value), min_value)
+
+
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
+        self.comfort_class = int(limitation(comfort_class, 1, 7))
+        self.clean_mark = int(limitation(clean_mark, 1, 10))
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self,
+                 distance_from_city_center: float,
+                 clean_power: int,
+                 average_rating: float,
+                 count_of_ratings: int) -> None:
```

Comment:
The best approach of placing parameters is to move them on the next line after function name